alpha-api/src/main/java/at/ac/tuwien/kr/alpha/api/programs/literals/Literal.java

```diff
@@ -0,0 +1,39 @@
+package at.ac.tuwien.kr.alpha.api.programs.literals;
+
+import java.util.List;
+import java.util.Set;
+
+import at.ac.tuwien.kr.alpha.api.grounder.Substitution;
+import at.ac.tuwien.kr.alpha.api.programs.Predicate;
+import at.ac.tuwien.kr.alpha.api.programs.atoms.Atom;
+import at.ac.tuwien.kr.alpha.api.terms.Term;
+import at.ac.tuwien.kr.alpha.api.terms.VariableTerm;
+
+/**
+ * A literal according to the ASP Core 2 Standard.
+ * Wraps an {@link Atom} that may or may not be negated.
+ *
+ * Copyright (c) 2017-2021, the Alpha Team.
+ */
+// TODO go through implementations and pull out stuff that can be default-implemented here
```

Comment:
Is this TODO planned to be addressed in this PR?

Reply:
No, this should be addressed as part of sub-item "Revisit and potentially refactor Atom and Literal hierarchies" of #300 and should be left in there until such a time.